src/server/pkg/deploy/assets/assets.go

```diff
@@ -63,7 +63,7 @@ func PachdRc(shards uint64, backend backend, hostPath string, version string) *a
 	// we turn metrics on only if we have a static version this prevents dev
 	// clusters from reporting metrics
 	metrics := "true"
-	if version == "" {
+	if version == "local" {
```

Comment:
Could we have this be a constant that's stored somewhere rather than a magic value.
I feel like that would prevent us getting in the same situation as before where we changed the value in one place but not another.
Probably locate it in `src/server/pkg/deploy/deploy.go`